script.js

```diff
@@ -1 +1,25 @@
-console.log('this works');
\ No newline at end of file
+// Set 1
+
+var div;
+
+// create a buildBox function
+function buildBox() {
+	// create and insert an empty div on the page with: positioning, top and left coordinates, background color expressed as rgba 
+	div = document.createElement('div');
```

Comment:
We shouldn't limit this function to modifying one particular existing variable in the global scope, rather create a new variable to work with inside this function, and return it at the end. Assume we're going to use this function to make many divs that we will want to work with in the global scope.